lampod-cli/src/main.rs

```diff
@@ -44,15 +48,57 @@ fn main() -> error::Result<()> {
     Ok(())
 }
 
+fn write_words_to_file<P: AsRef<Path>>(path: P, words: &Option<String>) -> io::Result<()> {
+    let mut file = OpenOptions::new()
+        .write(true)
+        .create(true)
+        .truncate(true)
+        .open(path)?;
+
+    match words {
+        Some(ref value) => {
+            file.write_all(value.as_bytes())?;
+        }
+        None => {
+            file.write_all(b"")?;
+        }
+    }
+
+    Ok(())
+}
+
+fn load_words_from_file<P: AsRef<Path>>(path: P) -> io::Result<Option<String>> {
+    let mut file = File::open(path)?;
+    let mut content = String::new();
+
+    file.read_to_string(&mut content)?;
+
+    if content.is_empty() {
+        Ok(None)
+    } else {
+        Ok(Some(content))
+    }
+}
+
 /// Return the root directory.
 fn run(args: LampoCliArgs) -> error::Result<()> {
+    let path = std::env::home_dir().expect("Impossible to get the home directory");
```

Comment:
```suggestion
    let path = std::env::home_dir().ok_or(error::anyhow!("Impossible to get the home directory"))?;
```

In lampo we never panic